OutOfSchool/OutOfSchool.DataAccess/Repository/SensitiveEntityRepositorySoftDeleted.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.Services.Repository;
+
+public class SensitiveEntityRepositorySoftDeleted<T> : EntityRepositorySoftDeleted<Guid, T>, ISensitiveEntityRepositorySoftDeleted<T>
```

Comment:
(Just wondering) Is there any necessity for this specific class? 
It adds no logic, and reuses `EntityRepositorySoftDeleted` in whole.

Reply:
I agree. But we have the same approach with the SensitiveEntityRepository and I used it.

Reply:
Closing this thread as possible changes goes out of the task scope. 
At same time I'm not convinced that usage of inheritance is beneficial here.